### ISSUE TITLE
Initialization conditions of pre-trained neural networks？

### ISSUE THREAD

**Issue by YouqiongLiu** [isds-neu/PhyCRNet#18]
Hello,

Thanks a lot for this great work that you published transparently.

Pretrain the network is from a small number of time steps (e.g., 100) and then gradually train on a longer dynamical evolution. Taking 2D Burgers' equation as an example, we pretrain the model from 100, then 200 and 500, and finally 1000. 

I have a questions anout the initialization condition.

For pretrained networks with different time steps(e.g. 100, 200, 500, 1000), is the initialization condition always 'uv0 = uv[0:1,...] 
    input = torch.tensor(uv0, dtype=torch.float32).cuda()'?

If I keep training from t=0 for different time steps, it turns out to have extremely large loss for 500 timesteps.
[19998/20000 99%] loss: 0.1154880226
[19999/20000 99%] loss: 0.1154880226
[20000/20000 100%] loss: 0.1154880226
The predicted error is:  0.12718813
